README.md

```diff
@@ -48,6 +48,16 @@ semantics instead. In this mode, the model will be updated in the cache as well
 Kasket.setup(write_through: true)
 ```
 
+#### Events Callback
+
+You can configure a callable object to listen to events, e.g. `cache_hit`. This can be useful to emit metrics and observe Kasket's behaviour.
```

Comment:
I think the Readme should contain a full list of callback events (which currently is just `cache_hit`). The “e.g. `cache_hit`” sounds like more events exist but are deliberately not documented.

Reply:
That's fair, thanks. I'll add them.